setup.cfg

```diff
@@ -120,6 +120,7 @@ extend-ignore =
     W605,
     RST210,
     RST213,
+    RST305,
```

Comment:
I forgot to add this error code as one of the linter checks to ignore. This one checks for undefined substitutions, but isn't smart enough to check for our globally defined substitutions.